lisa/operating_system.py

```diff
@@ -1119,6 +1136,22 @@ def _get_information(self) -> OsInformation:
 
         return information
 
+    def add_azure_core_repo(self) -> None:
+        # there is no available repo for distro which higher than ubuntu 18.04,
+        # use bionic for temp solution
+        if self.information.version >= "18.4.0":
```

Comment:
The code was written in 3 years ago. We should check other versions, and add support for 20.04, and >=22.04 as default.